### PR TITLE
Update capz membership: Add shysank to maintainers. jackfrancis, jsturtevant to pms

### PR DESCRIPTION
**Pull request by shysank**

### CODE REVIEW

**Comment by shysank** [comment]
cc @CecileRobertMichon

File: config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml [hunk 1]
```diff
@@ -87,6 +88,8 @@ teams:
     - CecileRobertMichon
     - cpanato
     - devigned
+    - jackfrancis
+    - jsturtevant
     - juan-lee
```
Comment (CecileRobertMichon) [inline config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml:93]: can you please also remove juan-lee here and nader-ziada above? ref https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/ccd08ea27a3f20cbf49ed2b1ecdf6744b3671c93

Comment (shysank) [reply config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml:93]: should I add myself to cluster-api-provider-azure-admins as well?

Comment (CecileRobertMichon) [reply config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml:93]: yes

Comment (shysank) [reply config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml]: done

**Pull request merged**